src/components/AllReviews.jsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from "react";
+import { fetchReviews } from "../api";
+
+const AllReview = () => {
+  const [allReviews, setAllReviews] = useState([]);
+  const [page, setPage] = useState("1");
+  const [next, setNext] = useState("");
+  const [previous, setPrevious] = useState("");
+  useEffect(() => {
```

Comment:
We should have a loading state for our fetch requests to show users that the page is loading whilst the data is being fetched.